internal/semantic/version-cran.go

```diff
@@ -0,0 +1,46 @@
+package semantic
```

Comment:
Is there an upstream documentation page for the versioning that we can link from here?